pyipmi/ipmitool.py

```diff
@@ -438,7 +438,7 @@ def usage(toplevel=False):
   -h               Show this help
   -v               Be verbose
   -V               Print version
-  -I <interface>   Set interface (available: rmcp, aardvark, ipmitool)
+  -I <interface>   Set interface (available: rmcp, aardvark, ipmitool, ipmbdev)
```

Comment:
theres a description for the parameters missing. the parameters  (ipmb device path) are missing in general ;)

Reply:
done